### PR TITLE
LRQA-58374, LRQA-59123 | master

### PR DESCRIPTION
**Pull request by zhangyan0701**
https://issues.liferay.com/browse/LRQA-58374
https://issues.liferay.com/browse/LRQA-59123

This is for master only, thanks!

/cc @john-co

### CODE REVIEW

**Comment by zhangyan0701** [comment]
ci:test:sf

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:sf

**Comment by zhangyan0701** [comment]
ci:test:sf

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by john-co** [comment]
Hi @zhangyan0701, Can you double check locally if we still need the reinitialize commit? There were some fixes that went in the past few days that may have fixed it.

**Comment by zhangyan0701** [comment]
Resend in [#191](https://github.com/zhangyan0701/liferay-portal/pull/191)

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/contentpage/ContentPages.macro [hunk 1]
```diff
@@ -4,11 +4,15 @@ definition {
 		if (isSet(collectionName)) {
 			Navigator.gotoNavNested(navNested = "${collectionName}");
 
+			WaitForLiferayEvent.initializeLiferayEventLog();
```
Comment (jbalsas) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/contentpage/ContentPages.macro:7]: I see we're still doing this... can we prioritze improving the way we collect and analyze events to remove all additional initializations, please?

Comment (zhangyan0701) [reply portal-web/test/functional/com/liferay/portalweb/macros/layout/contentpage/ContentPages.macro:7]: Hi @jbalsas,
I added this to fix IE11 failures when adding pages. This error blocks another ticket, so used it as a temporary solution to make the tests keep running after adding page. I will investigate more about how to fix this in a better way and then remove the initialization, but I am afraid that may take some time.

Regards!

Comment (jbalsas) [reply portal-web/test/functional/com/liferay/portalweb/macros/layout/contentpage/ContentPages.macro:7]: That's okay @zhangyan0701... just feels like one of those things that never get done so we're always walking on eggshells over a flaky test infrastructure and setup.

Let's try to put some time into this when we can!

**Pull request closed without merging**